R/droplet-ssh.R

```diff
@@ -108,8 +108,14 @@ droplet_ip <- function(x) {
     stop("No network interface registered for this droplet\n  Try refreshing like: droplet(d$id)",
       call. = FALSE)
   }
-
-  v4[[1]]$ip_address
+  ips = do.call("rbind", lapply(v4, as.data.frame))
+  public_ip = ips$type == "public"
+  if (!any(public_ip)) {
+    ip =v4[[1]]$ip_address
```

Comment:
make sure there's spaces on each side of operators

Reply:
Used styler to fix